modules/offers/server/controllers/offers.server.controller.js

```diff
@@ -608,6 +608,21 @@ exports.list = function (req, res) {
     }
   }
 
+  // offer maxGuests filter
+  if (filters.hasObjectFilter('offer')) {
```

Comment:
Should we avoid running the query if value is 1? Would speed things up in default cases.